server/app.js

```diff
@@ -25,6 +25,13 @@ const dist = path.join(__dirname, '/../client/dist');
 
 
 
+// This line sets the environment variables, since we are on our local machines
+// Therefore, in production (or whenever we are hosted on an actual server),
+//   this line can be removed along with the .env file
+require('dotenv') // same as const dotenv = require('dotenv');
+  .config(); // we just want to call .config, not save
+
```

Comment:
This is unnecessary, as it exists above, where it is needed, but having it also be here should not be a breaking change, so I'll let it slide...